CHANGELOG.md

```diff
@@ -1,6 +1,28 @@
+# dbt_unified_rag v0.1.0-a6
```

Comment:
Apologies I didn't realize this before. Be sure to update the install version in the README with the new -a6 version.

Reply:
Good catch!